ethportal-api/src/types/consensus/light_client/bootstrap.rs

```diff
@@ -152,4 +153,39 @@ mod test {
         LightClientBootstrap::from_ssz_bytes(&expected, ForkName::Capella).unwrap();
         assert_eq!(content.as_ssz_bytes(), expected);
     }
+
+    #[rstest]
+    #[case("case_0")]
+    #[case("case_1")]
+    fn serde_light_client_bootstrap_deneb(#[case] case: &str) {
```

Comment:
Am I reading this right that these are just two randomly generated examples? They aren't some specific variants being tested?

If it's random, then I guess I have no notes. But if there's a particular difference being tested, it would be nice to make that clearer.

Reply:
They are random.